src/api/client.go

```diff
@@ -115,9 +146,50 @@ func (c *Client) get(endpoint string) (*http.Response, error) {
 		return nil, err
 	}
 
+	if contentType != "" {
+		req.Header.Add("Content-Type", contentType)
+	}
+
 	return c.HTTPClient.Do(req)
 }
 
+// DeleteJSON makes a DELETE request to an endpoint. Adds Content-Type application/json
+func (c *Client) DeleteJSON(endpoint string, v url.Values, obj interface{}) error {
+	csrf, err := c.CSRF()
+	if err != nil {
+		return err
+	}
+
+	endpoint = strings.TrimLeft(endpoint, "/")
+	endpoint = c.Addr + endpoint
+
+	// add parameters to url
```

Comment:
unnecessary comment. sometimes over commenting is bad as the issue with comments is that they need to be maintained, not talking about this one but in general. 

So comments are usually appreciated when you are doing something logical which might not be clear to someone at first glance.